src/resolveElements.js

```diff
@@ -45,7 +45,7 @@ function createElements(match, Components, matchData) {
 
     if (!isResolved(Component) || !isResolved(data)) {
       // Can't render.
-      return null;
+      return undefined;
```

Comment:
This matches Relay, where returning `undefined` in `render` will keep rendering the last view